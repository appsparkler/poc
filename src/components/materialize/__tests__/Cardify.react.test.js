import React, {Component} from 'react'
import cardify from '../Cardify.react'
import renderer from 'react-test-renderer'
// import {configure, shallow} from 'enzyme'
// import Adapter from 'enzyme-adapter-react-16'

// configure({adapter: new Adapter()})

it('should correctly cardify a component', () => {
  class TestComponent extends Component {
    render() {
      return (
        <div>Test Component for cardify hoc component</div>
      )
    }
  }
  const CardifiedComponent = cardify(TestComponent)
  // const wrapper = shallow(<CardifiedComponent />)
  const wrapper = renderer.create(<CardifiedComponent />)
  const tree = wrapper.toJSON()
  expect(tree).toMatchSnapshot()
})
