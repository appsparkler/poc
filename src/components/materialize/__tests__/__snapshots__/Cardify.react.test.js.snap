// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly cardify a component 1`] = `
<div
  className="card"
>
  <div
    className="card-content"
  >
    <div>
      Test Component for cardify hoc component
    </div>
  </div>
</div>
`;
